package.json

```
@@ -119,7 +119,7 @@
     "stylelint-prettier": "^2.0.0"
   },
   "devDependencies": {
-    "lerna": "^4.0.0",
+    "lerna": "^5.0.0",
```

Maybe we could set the version to `^5.4.2` here for consistency with the new version resolved in `yarn.lock`?

```suggestion
    "lerna": "^5.4.2",
```

---

Actually a `5.4.3` was released a couple of hours ago and we could also update to it: https://github.com/lerna/lerna/releases/tag/v5.4.3